src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Value.ReadTests.cs

```diff
@@ -546,7 +551,6 @@ public static void TimeSpan_Read_KnownDifferences()
         [InlineData("00:00:60")]
         [InlineData("00:00:00.00000009")]
         [InlineData("900000000.00:00:00")]
-        [InlineData("1:00:00")] // 'g' Format
```

Comment:
Could you also add an empty string here just so we exercise the lower bound side of size checks?

Reply:
Sure done.